course_catalog/etl/youtube.py

```diff
@@ -28,6 +28,7 @@
 CONFIG_FILE_FOLDER = "youtube"
 YOUTUBE_API_SERVICE_NAME = "youtube"
 YOUTUBE_API_VERSION = "v3"
+WILDCARD_PLAYLIST_ID = "wildcard"
```

Comment:
* breaks the YAML parser which insists on alpha numeric data. Maybe there is a better name for this?